plenum/test/consensus/view_change/test_sim_view_change.py

```diff
@@ -133,7 +134,10 @@ def check_view_change_completes_under_normal_conditions(random: SimRandom,
     committed_above_cp = [c for c in committed if c.pp_seq_no > stable_checkpoint]
     for n in pool.nodes:
         if n._data.stable_checkpoint >= stable_checkpoint:
-            assert committed_above_cp == n._data.preprepared[:len(committed_above_cp)]
+            for expected_batch, actual_batch in zip(committed_above_cp, n._data.preprepared[:len(committed_above_cp)]):
+                assert expected_batch.pp_view_no == actual_batch.pp_view_no
```

Comment:
Why did we replace it? Do we see that `pp_original_view_no` is different? If so, this looks suspicious.

Reply:
As far as I understand it `pp_view_no` is playing role of original view no, and `view_no` is current view no. When debugging those tests I found that sometimes more than one view change was triggered, so `view_no` wasn't 1 as expected, but rather something like 2, 3, and in some cases I've seen numbers like 6. But in all cases `pp_view_no` stayed at 0, as expected.